### ISSUE TITLE
Language problem

### ISSUE THREAD

**Issue by esomola** [WPprodigy/woocommerce-external-product-embed#40]
My external woocommerce shop is in Spanish.

When I display the products on an external website it shows the buttons VIEW PRODUCT and SALE in English.

How can I solve this?

**Comment by WPprodigy** [comment]
Yeah this is partly my bad, the plugin needs to be internationalized better.

To change the button text, you can pass an attribute to the shortcode like this: `[wcepe_products orderby='title' order='desc' number='12' columns='4' button='Example Text Here']`

The on-sale text though you will currently need to override the template and edit this line: https://github.com/WPprodigy/woocommerce-external-product-embed/blob/master/templates/product-content.php#L65
